web/src/components/organisms/Project/Content/ViewsMenu/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from "react-router-dom";
+
+import ViewFormMobal from "@reearth-cms/components/molecules/View/ViewFormModal";
+import ViewsMenuMolecule from "@reearth-cms/components/molecules/View/viewsMenu";
+
+import useHooks from "./hooks";
+
+const ViewsMenu: React.FC = () => {
+  const { modelId } = useParams();
+
+  const {
+    views,
+    handleViewModalOpen,
+    // handleViewUpdateModalOpen,
+    handleViewRenameModalOpen,
+    selectedView,
+    viewModalShown,
+    handleViewModalReset,
+    handleViewCreate,
+    handleViewUpdate,
+    handleViewDelete,
+    handleViewDeletionModalClose,
+  } = useHooks({ modelId });
+
+  return (
+    <>
+      <ViewsMenuMolecule
+        views={views}
```

Comment:
This is giving a type error because you are not converting from the GraphQL type.